index.html

```diff
@@ -6558,13 +6558,13 @@ <h2 id="defining-a-valid-audioworkletprocessor">
               <ul>
                 <li>Nodes that transform their inputs, and are active only
                 while connected inputs and/or script references exist. Such
-                nodes should return <code>false</code> from
+                nodes SHOULD return <code>false</code> from
```

Comment:
I'd say SHOULD is completely appropriate here since we're offering the user options on how to employ the procedure